src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.scss

```diff
@@ -9,9 +9,7 @@
 
     ::ng-deep {
         .ui-toolbar {
-            box-shadow: $md-shadow-1;
-            height: 60px;
-            margin-bottom: 2px;
+            height: 48px;
```

Comment:
What?

Reply:
there is no need of the shadow and margin according to the new design.